internal/globpath/globpath_test.go

```diff
@@ -1,3 +1,7 @@
+// +build !windows
+
+// TODO: should be enabled for Windows when Glob related issues for Windows are fixed
```

Comment:
@ssoroka Yes, super asterisk doesn't work for Windows, probably because there is bug in `internal/globpath/globpath.go`.

I put more information here: https://github.com/influxdata/telegraf/issues/6248#issuecomment-731645551

Fixing that issue in `internal/globpath/globpath.go` will enable super asterisk for Windows for these plugins:
- file
- filecount
- filestat
- logparser
- phpfpm
- tail